app/src/main/java/com/github/neighbortrader/foodboardapp/requests/CreateNewUserRequest.java

```diff
@@ -0,0 +1,64 @@
+package com.github.neighbortrader.foodboardapp.requests;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.github.neighbortrader.foodboardapp.clientmodel.User;
+
+import java.util.List;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class CreateNewUserRequest extends AsyncRequest<Void> {
```

Comment:
Diese Klasse ist veraltet und nicht in Verwendung. Wird mit dem nächsten MergeRequest für #12 #7